docs/profiling.md

```diff
@@ -9,6 +9,8 @@ Profiling is active by default. The recording can be triggered in to different w
   
 That should flood your xhgui with requests.
 
+hint: if `PHP_DEBUGGER` is set to `xdebug`, then the xh-profiler won't profile anything 
```

Comment:
```suggestion
hint: if xdebug is enabled, then the xh-profiler won't profile anything 
```

Reply:
i can have both xdebug and the profiler active at the same time. xhprof is disabled through the env-var and not xdebug in itself